Week 1 Quiz Answers

```diff
@@ -0,0 +1,112 @@
+HTML - Brittany Branson's Answers
+
+Question #1
+
+What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
+
+//Semantics is the meaning of a word beyond it's dictionary definition.
+Regarding HTML, semantic tags tell both the developer and browser what it and it's contents mean or contains, rather than just how it is presented or stylized.
+For example, <h1> tag is semantic, in that a developer and browser will immediately understand what kind of information the <h1> tag will contain and present.
+The <p> tag is both semantic and presentational, because browsers and developers not only understand what information will be contained within the <p> tag, but a browser generally understands how to present the tag (even without more specific styling).
+The <i> tag is not semantic, for it is a style. Simply seeing the <i> tag will not tell a developer or browser any more information other than that information will be italicized.
+Furthermore, it is not semantic because even italicized words could have different meanings (i.e. a book title vs. emphasis).
+
+Question #2
+
+What is the purpose of the alt attribute? e.g:
+
+<img src='https://jesse.sh/img/me.jpg' alt='Jesse Shawl'/>
+
+//Alt tags are useful for both developers and users. For developers, the alt attribute aids him/her as he/she reads/visualizes code, for it gives an image a title and will display that title even if the image fails to load in the browser.
+It is especially useful for users with disabilities such as blindness. Certain text readers and other tools will say the alt attribute out loud, permitting a user with a disability to understand that an image is located on the page.
```

Comment:
:+1:  its also useful more so for users if the image fails not just developers in testing code :)